File: app/src/main/java/com/duy/pascal/backend/exceptions/grouping/GroupingException.java
package com.duy.pascal.backend.exceptions.grouping;

import com.duy.pascal.backend.exceptions.ParsingException;
import com.duy.pascal.backend.linenumber.LineInfo;


public class GroupingException extends ParsingException {

    public GroupingException(LineInfo line, String message) {
        super(line, message);
    }

    public GroupingException(LineInfo line) {
        super(line);
    }
}

File: libCompiler/src/main/java/com/duy/pascal/backend/exceptions/grouping/GroupingException.kt
package com.duy.pascal.backend.exceptions.grouping

import com.duy.pascal.backend.exceptions.ParsingException
import com.duy.pascal.backend.linenumber.LineInfo


open class GroupingException : ParsingException {

    constructor(line: LineInfo, message: String) : super(line, message) {}

    constructor(line: LineInfo) : super(line) {}
}
